service/src/test/java/skills/intTests/SkillApprovalSpecs.groovy

```diff
@@ -726,14 +742,17 @@ class SkillApprovalSpecs extends DefaultIntSpec {
         List<String> performedAfter = userPerformedSkillRepo.findAll().collect { "${it.projectId}-${it.skillId}_${it.userId}" }
 
 
-        List<String> expectedIds = [
-                "TestProject1-skill1_${users[0]}",
-                "TestProject1-skill1_${users[1]}",
-                "TestProject1-skill1_${users[2]}",
-                "TestProject1-skill1_${users[3]}",
-                "TestProject1-skill1_${users[4]}",
-                "TestProject1-skill1_${users[5]}",
-        ].sort()
+        List<String> expectedIds = (0..5).collect {
+            "TestProject1-skill1_${users[it]}"
+        }.sort()
+//        [
+//                "TestProject1-skill1_${users[0]}",
+//                "TestProject1-skill1_${users[1]}",
+//                "TestProject1-skill1_${users[2]}",
+//                "TestProject1-skill1_${users[3]}",
+//                "TestProject1-skill1_${users[4]}",
+//                "TestProject1-skill1_${users[5]}",
+//        ].sort()
```

Comment:
should these commented out lines be deleted?

Reply:
yes, removed!